bin/build-frontend-static.js

```diff
@@ -0,0 +1,19 @@
+const globWatch = require('glob-watcher')
+const fs = require('fs-extra')
+
+async function main(watch=false) {
+  const copy = async () => {
+    await fs.copy('./static/', './html-dist/')
+    console.log('- copied files from static folder')
+  }
+  await copy()
+
+  if (watch) {
+    // (whats the usecase of this? live update after pulling in translations??)
```

Comment:
there aren't any translations in the static folder so I guess this comment doesn't belong here.

Reply:
Don't think that I wrote that comment. But can remove it for sure.